src/components/Tables/TasksTable.jsx

```diff
@@ -47,15 +47,19 @@ function TasksTable({ tasks, height }) {
               );
             })} */}
           {tasks &&
-            [...tasks]
-              .slice(start, rowsPerPage())
+            tasks
+              .slice(start, start + rowsPerPage())
```

Comment:
@Ddupasquier This was the root issue -- without this change it was trying to do e.g. `.slice(18, 18)` or `.slice(36, 18)`, which returns an empty array.

Reply:
Omg, what a silly oversight xD